DigitalLearningSolutions.Web/Views/NotificationPreferences/Index.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+
+@model NotificationPreferencesViewModel
+
+@{
+  ViewData["Title"] = "Notification Preferences";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading">Notification Preferences</h1>

```

Comment:
We decided to use explicit typography classes on headings, so could we also specify the typography class explicitly on this h1 (besides the id)  https://service-manual.nhs.uk/design-system/styles/typography#headings